client/rest/src/plugins/metadata/metal.js

```diff
@@ -0,0 +1,230 @@
+const catapult = require('../../catapult-sdk');
+const { sha3_256 } = require('@noble/hashes/sha3');
+const bs58 = require('bs58');
+
+const METAL_ID_HEADER_SIGNATURE = [0x0B, 0x2A];
+const METAL_ID_LENGTH = 34;
+
+const metal = {
+	/**
+	 * convert a metal id (e.g. 'FeF65JftVPEGwaua35LnbU9jK46uG3W8karGDDuDwVEh8Z') into its associated composite hash.
+	 * @param {string} metalId metal id to convert.
+	 * @returns {Uint8Array} Associated composite hash.
```

Comment:
I think there was miscommunication somewhere 😅 

In javadoc, it's correct to capitalize first word in descriptions (e.g. Convert a ..., Metal id ..., etc) [what you had before]

Reply:
sorry, i made mistake🙏
maybe, i fixed